src/docs/patterns/user-auth.mdx

```diff
@@ -0,0 +1,324 @@
+        'Screens in the registration workflow.',
+        '1: Enter email address',
+        '2: Accept EULA',
+        '3: Verify email',
+        '4: Create password',
+        '5: Add account details',
+        '6: Success: account created',
+    ]}
+/>
+<ImageGrid images={[RegistrationDesktop]} caption={'A registration screen example on desktop.'} regularWidth/>
+
+## Joining an Organization
+
+Some applications may require users to join specific groups or organizations after creating an account. Generally, there are three ways for a user to join an organization:
+
+**Request Access**
+
+This occurs when a user has an account but need to access an invitation-only organization. This workflow involves the following steps:
+
+-   Enter organization identifier
+-   Success: request sent
+-   Receive an email invitation
+-   Success: organization joined
+
+**Enter Passcode**
+
+Joining by passcode occurs when anyone with an organization identifier and passcode can join an organization. This workflow includes the following steps:
+
+-   Enter organization identifier
+-   Enter organization passcode
+-   Success: organization joined
+
+**Join with Invitation**
+
+Joining via invitation occurs when a system admin extends an invitation for a user to join a group. If the user does not yet have an account, they will first proceed through the invitation-based registration workflow followed by the screens for joining the organization. This workflow includes the following steps:
+
+-   Receive an email invitation
+-   Success: organization joined
+
+### Screens
+
+The majority of the joining-an-organization screens are the same regardless of the underlying mechanism. Use the following screens for the joining-an-organization workflow where appropriate.
+
+<ImageGrid
+    images={[EnterOrganizationCode, EnterOrganizationPasscode, EnterOrganizationConfirm, EnterOrganizationWelcome]}
+    caption={[
+        'Screens in the joining-an-organization workflow.',
+        '1: Enter organization identifier',
+        '2: Enter organization passcode',
+        '3: Success: request sent',
+        '4: Success: organization joined',
+    ]}
+/>
+<ImageGrid images={[JoinAnOrganizationDesktop]} caption={'A join-an-organization screen example on desktop.'} regularWidth />
+
+## Login
+
+<!--
+An NFC login might either be documented here, or wait until we start adding in device registery / discovery pattern
+-->
+
+Logging in can usually be accomplished in one of two ways: basic login or login via a third-party.
+
+**Basic Login**
+
+A simple login presents fields for the users to enter their credentials and a button to submit them for verification.
+
+The login screen should always offer some workaround if a user has some trouble getting authorized, either because they don't have an account, or that they forget their credentials. In your login screen, there should be buttons that can initiate those extra functions, such as Registration and Forgot Password (see below).
```

Comment:
```suggestion
The login screen should always present information on how to create an account and what to do if a user has forgotten their password — these buttons should trigger the Registration and Forgot Password (see below) workflows .

> If you are using invitation-based registration, you should still provide information on how to request an invite.
```

Reply:
Although we currently don't have designs for it yet — should we take into account the fact that the user might forget their email as well?

Reply:
Sure, we can say "Credentials" instead of password to cover both